main.css

```diff
@@ -0,0 +1,21 @@
+.hidden {
+	display:none;
+}
+
+body {
+	padding:20px;
+}
+
+button {
+	display:block;
+	margin-top:10px;
+}
```

Comment:
Consider placing 'body' and 'button at the top of the doc (before the class 'hidden') since they affect the global layout.